sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2508,6 +2508,14 @@ object functions {
   def current_date(): Column = withExpr { CurrentDate() }
 
   /**
+   * Returns the current date in the given timezone as a date column.
+   *
+   * @group datetime_funcs
+   * @since 2.3.0
+   */
+  def current_date(timeZone: String): Column = withExpr { CurrentDate(Option(timeZone)) }
```

Comment:
s/`Option`/`Some`

Reply:
What if someone pass a `null` as value? `Option` is safer in this case...